src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -582,6 +584,40 @@ public final Response getEventBookingForGivenGroup(@Context final HttpServletReq
         }
     }
 
+    /** gets a list of event bookings for all groups owned.
+     *
+     */
+    @GET
+    @Path("{event_id}/groups_bookings")
+    @Produces(MediaType.APPLICATION_JSON)
+    @GZIP
+    @ApiOperation(value = "List event bookings for a specific event")
+    public final Response getEventBookingForAllGroups(@Context final HttpServletRequest request,
+                                                       @PathParam("event_id") final String eventId) {
+        try {
+            RegisteredUserDTO currentUser = userManager.getCurrentRegisteredUser(request);
+            List<EventBookingDTO> eventBookings = this.mapper.mapAsList(bookingManager.getBookingsByEventId(eventId), EventBookingDTO.class);
+
+            if (!isUserTeacherOrAbove(userManager, currentUser)) {
+                return new SegueErrorResponse(Status.FORBIDDEN, "You do not have permission to use this endpoint.").toResponse();
```

Comment:
There's already `SegueErrorResponse.getIncorrectRoleResponse()` (or something) for this. Not sure it's worth changing this now, but for reference!